src/LongRunning/Gapic/OperationsGapicClient.php

```diff
@@ -166,14 +166,13 @@ private static function getGapicVersion()
      *           A `ChannelCredentials` object for use with an SSL-enabled channel.
      *           Default: a credentials object returned from
      *           \Grpc\ChannelCredentials::createSsl()
-     *           NOTE: if the $channel optional argument is specified, then this option is unused.
+     *           NOTE: if the $channel optional argument is specified, then this argument is unused.
      *     @type bool $forceNewChannel
      *           If true, this forces gRPC to create a new channel instead of using a persistent channel.
      *           Defaults to false.
      *           NOTE: if the $channel optional argument is specified, then this option is unused.
      *     @type \Google\Auth\CredentialsLoader $credentialsLoader
-     *                              A CredentialsLoader object created using the
-     *                              Google\Auth library.
+     *           A CredentialsLoader object created using the Google\Auth library.
      *     @type array $scopes Required. A string array of scopes to use when acquiring credentials.
```

Comment:
Since the new proto comments switch from `array` to `typep[]`, shall we do the same in the GAPIC layer?
Not necessary in this PR

Reply:
Yes, good call. Created an issue for this here: https://github.com/googleapis/toolkit/issues/1533